.github/workflows/semesterly.yml

```diff
@@ -0,0 +1,74 @@
+name: Semesterly CI
+
+on:
+  push:
+    branches: [ master, staging, cs310-spring21, github-actions ]
+  pull_request:
+    branches: [ master, staging, cs310-spring21, github-actions ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:10.8
```

Comment:
I think we're using postgres:12.1. I'm not if that can be changed, but just a note!